### ISSUE TITLE
This bundle is not maintained anymore.

### ISSUE THREAD

**Issue by ghost** [misd-service-development/guzzle-bundle#90]
Please add a warning that this bundle is not maintained anymore.

Reasons:
- There are newer versions of Guzzle.
- There are other bundles. (rel #28)

**Comment by mablae** [comment]
Oh, thanks for mentioning that! 

@JHGitty Could you please share the more up2date Symfony2 Guzzle Bundles?

**Comment by ghost** [comment]
Here you can find the latest updated bundles:
https://github.com/search?o=desc&q=guzzle+bundle&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93

**Comment by garak** [comment]
+1. Please also mark project as abandoned on packagist

**Comment by Tobion** [comment]
I agree. @thewilkybarkid can you please mark the project as abandoned on git and packagist?

**Comment by thewilkybarkid** [comment]
Sorted!

**Comment by Tobion** [comment]
Maybe close all issues and PRs?
